src/libraries/tick_math.cairo

```diff
@@ -22,9 +22,14 @@ mod TickMath {
     }
 
     /// The minimum value that can be returned from `get_sqrt_ratio_at_tick`. Equivalent to get_sqrt_ratio_at_tick(MIN_TICK).
-    const MIN_SQRT_RATIO: u256 = 4295128739;
+    fn MIN_SQRT_RATIO() -> FixedType {
+        FixedTrait::new(4295128739, false)
+    }
+
     /// The maximum value that can be returned from `get_sqrt_ratio_at_tick`. Equivalent to get_sqrt_ratio_at_tick(MAX_TICK).
-    const MAX_SQRT_RATIO: u256 = 1461446703485210103287273052203988822378723970342;
+    fn MAX_SQRT_RATIO() -> FixedType {
+        FixedTrait::new(1461446703485210103287273052203988822378723970342, false)
+    }
```

Comment:
Are we sure they represent the same thing? Why was this change necessary? Should we add something like MIN/MAX_SQRT_RATIO_X96 for clarity?

Reply:
I agree this could be useful specially for the future when we will do a refactor. We still need to add the 128.128 type.

Reply:
✅